test/Microsoft.TemplateEngine.EndToEndTestHarness/BuiltInTemplatePackagesProviderFactory.cs

```diff
@@ -0,0 +1,69 @@
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Abstractions.PhysicalFileSystem;
+using Microsoft.TemplateEngine.Abstractions.TemplatePackage;
+using Microsoft.TemplateEngine.Utils;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.TemplateEngine.EndToEndTestHarness
```

Comment:
should we use default factory instead?

Reply:
I changed to use DefaultProvider implementation but I think its fine to have different factory.